<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrowserWidget</class>
 <widget class="QWidget" name="BrowserWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolBar" name="toolBar">
     <property name="windowTitle">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <addaction name="actionBrowserBack"/>
     <addaction name="actionBrowserForward"/>
     <addaction name="actionBrowserRefresh"/>
     <addaction name="actionBrowserStop"/>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtUrl"/>
   </item>
   <item>
    <widget class="CustomWebView" name="webView" native="true"/>
   </item>
  </layout>
  <action name="actionBrowserBack">
   <property name="icon">
    <iconset resource="../rsrc/resources_autogenerated.qrc">
     <normaloff>:/icons/arrow-left</normaloff>:/icons/arrow-left</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="toolTip">
    <string>Back</string>
   </property>
  </action>
  <action name="actionBrowserForward">
   <property name="icon">
    <iconset resource="../rsrc/resources_autogenerated.qrc">
     <normaloff>:/icons/arrow-right</normaloff>:/icons/arrow-right</iconset>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
   <property name="toolTip">
    <string>Forward</string>
   </property>
  </action>
  <action name="actionBrowserRefresh">
   <property name="icon">
    <iconset resource="../rsrc/resources_autogenerated.qrc">
     <normaloff>:/icons/reload</normaloff>:/icons/reload</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionBrowserStop">
   <property name="icon">
    <iconset resource="../rsrc/resources_autogenerated.qrc">
     <normaloff>:/icons/x</normaloff>:/icons/x</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomWebView</class>
   <extends>QWidget</extends>
   <header>CustomWebView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../rsrc/resources.qrc"/>
 </resources>
 <connections/>
</ui>
